### PR TITLE
Add Continuation cleanup API

### PR DESCRIPTION
**Pull request by fengxue-IS**
Signed-off-by: Jack Lu <Jack.S.Lu@ibm.com>

### CODE REVIEW

**Comment by fengxue-IS** [comment]
Running personal build to verify changes, will mark ready when build/test passes

FYI @LinHu2016

**Comment by LinHu2016** [comment]
@fengxue-IS LGTM, since https://github.com/eclipse-openj9/openj9/pull/15611 has been merged, Could you please update VM_VMHelpers::cleanupContinuationObject() too, Thanks,
like
```
static VMINLINE void
cleanupContinuationObject(J9VMThread *vmThread, J9Object *objectPtr)
{
#if JAVA_SPEC_VERSION >= 19
	freeContinuation(vmThread, objectPtr);
#endif /* JAVA_SPEC_VERSION >= 19 */
}

```
not sure if freeContinuation() can be called directly from VM_VMHelpers.

**Comment by fengxue-IS** [comment]
> not sure if freeContinuation() can be called directly from VM_VMHelpers.

I think it will have to use the `vm->internalVMFunctions->`

**Comment by fengxue-IS** [comment]
@tajila can you take a look please

**Comment by fengxue-IS** [comment]
Fixed merge conflict with #15872

**Comment by tajila** [comment]
jenkins test sanity win,amac jdk19

File: runtime/vm/ContinuationHelpers.cpp [hunk 1]
```diff
@@ -147,6 +147,29 @@ yieldContinuation(J9VMThread *currentThread)
 	return result;
 }
 
+void
+freeContinuation(J9VMThread *currentThread, j9object_t continuationObject)
+{
+	J9VMContinuation *continuation = J9VMJDKINTERNALVMCONTINUATION_VMREF(currentThread, continuationObject);
+	if (NULL != continuation) {
```
Comment (tajila) [inline runtime/vm/ContinuationHelpers.cpp:154]: is it possible for this to be NULL

Comment (fengxue-IS) [reply runtime/vm/ContinuationHelpers.cpp:154]: if this continuation is freed after termination, GC can also call free when the continuation object is no longer referenced via the VMHelper wrapper.

**Pull request merged**